TweetClassifier/src/Database_Api.py

```diff
@@ -122,108 +121,105 @@ def update_collection(collection: Collection, new_values: [dict]) -> BulkWriteRe
         if 'class' in document:
             fields['$set']['class'] = document['class']
 
+        if 'trained' in document:
+            fields['$set']['trained'] = document['trained']
         requests.append(pymongo.UpdateOne({'id': document['id']}, fields, upsert=True))
 
-    result = collection.bulk_write(requests, ordered=False)
-    return result
+    validate = db_collection.bulk_write(requests, ordered=False)
```

Comment:
Why the rename here? The choice of result originally was to reflect that it is a BulkWriteResult object

Reply:
Was causing a shadowing issue, resolved in a different way now